flow/designs/gf12/ariane/config_hier_tmp.mk

```diff
@@ -0,0 +1,26 @@
+include $(dir $(DESIGN_CONFIG))/config.mk
```

Comment:
Why don't you just include config_hier.mk and unexport RTLMP_FLOW?  [for all tests]

Reply:
Makes sense... Will do that and and also change the FLOW_VARIANT

Reply:
Refactored .mk files to use include and set/unset appropriate variables